server/get_payload.go

```diff
@@ -41,12 +41,36 @@ var (
 	errFailedToConvert  = errors.New("failed to convert block from SSZ to JSON")
 )
 
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+type PayloadVersion string
+
+const (
+	PayloadV1 PayloadVersion = "V1"
+	PayloadV2 PayloadVersion = "V2"
```

Comment:
`PayloadV*` is too ambiguous in my opinion. Makes me think the payload itself is changing. Let's rename these to `GetPayloadV*` & rename the type to `GetPayloadVersion`. Later, we can rename this to `SubmitBlindedBlockV*`.